### ISSUE TITLE
[CLI] polymer init: Still can't init from application or element templates 

### ISSUE THREAD

**Issue by katejeffreys** [Polymer/tools#343]
From a fresh install of polymer-cli@next, `polymer init` fails to create application and element templates due to missing .gitignore in `node_modules/polymer-cli/templates/application/polymer-3.x`, etc. 

Related: https://github.com/Polymer/tools/pull/196

Downloading and pasting the `templates` folder from https://github.com/Polymer/tools/tree/master/packages/cli to replace my local `~/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/templates` folder fixes it. However, re-installing polymer-cli breaks it again.

Here's my console output, you can see that after a fresh install of polymer-cli@next, ~/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/templates/application/polymer-3.x does not have a `.gitignore` in it.

```
~ > which polymer
~ > ls -la /Users/katejeffreys/.nvm/versions/node/v9.11.1/bin/polymer
ls: /Users/katejeffreys/.nvm/versions/node/v9.11.1/bin/polymer: No such file or directory
~ > ls -la /Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/bin/
ls: /Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/bin/: No such file or directory
~ > ls -la /Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/templates/polymer-3.x
ls: /Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/templates/polymer-3.x: No such file or directory
~ > npm install -g polymer-cli@next
/Users/katejeffreys/.nvm/versions/node/v9.11.1/bin/polymer -> /Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/bin/polymer.js

> fsevents@1.2.3 install /Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/fsevents
> node install

... skip output of install command...

+ polymer-cli@1.7.0-pre.17
added 1740 packages in 52.512s
~ > which polymer
/Users/katejeffreys/.nvm/versions/node/v9.11.1/bin/polymer
~ > mkdir test-init
~ > cd test-init
~/test-init > polymer init
? Which starter template would you like to use? polymer-3-application
info: [init]    Running template polymer-3-application...
? Application name test-init
? Main element name test-init-app
? Brief description of the application 
error: [cli.main]   Uncaught exception: AssertionError [ERR_ASSERTION]: Trying to copy from a source that does not exist: /Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/templates/application/polymer-3.x/.gitignore
error: [cli.main]   AssertionError [ERR_ASSERTION]: Trying to copy from a source that does not exist: /Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/templates/application/polymer-3.x/.gitignore
    at EditionInterface.exports.copy (/Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/mem-fs-editor/lib/actions/copy.js:47:3)
    at EditionInterface.module.exports [as copyTpl] (/Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/mem-fs-editor/lib/actions/copy-tpl.js:10:8)
    at Polymer3ApplicationGenerator.writing (/Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/lib/init/application/application.js:122:21)
    at Object.<anonymous> (/Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/yeoman-generator/lib/index.js:417:23)
    at /Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/run-async/index.js:25:25
    at new Promise (<anonymous>)
    at /Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/run-async/index.js:24:19
    at /Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/yeoman-generator/lib/index.js:418:9
    at runCallback (timers.js:763:18)
    at tryOnImmediate (timers.js:734:5)
~/test-init > cd ..
~ > ls -la /Users/katejeffreys/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/templates/application/polymer-3.x/
total 48
drwxr-xr-x  10 katejeffreys  eng  320 May  6 19:18 .
drwxr-xr-x   5 katejeffreys  eng  160 May  6 19:18 ..
-rw-r--r--   1 katejeffreys  eng   14 Oct 26  1985 .npmignore
-rw-r--r--   1 katejeffreys  eng  804 Oct 26  1985 README.md
-rw-r--r--   1 katejeffreys  eng  708 Oct 26  1985 index.html
-rw-r--r--   1 katejeffreys  eng  179 Oct 26  1985 manifest.json
-rw-r--r--   1 katejeffreys  eng  284 Oct 26  1985 package.json
-rw-r--r--   1 katejeffreys  eng  104 Oct 26  1985 polymer.json
drwxr-xr-x   3 katejeffreys  eng   96 May  6 19:18 src
drwxr-xr-x   3 katejeffreys  eng   96 May  6 19:18 test
~ >
```

... after pasting the downloaded templates folder:

```
~ > cd test-init
~/test-init > polymer init
? Which starter template would you like to use? polymer-3-application
info: [init]    Running template polymer-3-application...
? Application name test-init
? Main element name test-init-app
? Brief description of the application 
   create package.json
   create index.html
   create manifest.json
   create polymer.json
   create README.md
   create src/test-init-app/test-init-app.js
   create test/test-init-app/test-init-app_test.html
   create .gitignore

Project generated!
Installing dependencies...
```

**Comment by justinfagnani** [comment]
I can't reproduce this and there are `.gitignore` files in the template directories.

**Comment by usergenic** [comment]
Fixed in master. #350

**Comment by usergenic** [comment]
Fix was released in polymer-cli@1.7.0
